taglib/mpeg/id3v2/id3v2framefactory.cpp

```diff
@@ -351,7 +351,7 @@ void FrameFactory::rebuildAggregateFrames(ID3v2::Tag *tag) const
       if(date.length() == 4) {
         tdrc->setText(tdrc->toString() + '-' + date.substr(2, 2) + '-' + date.substr(0, 2));
         if(tag->frameList("TIME").size() == 1) {
-          UnknownFrame *timeframe = static_cast<UnknownFrame *>(tag->frameList("TIME").front());
+          UnknownFrame *timeframe = dynamic_cast<UnknownFrame *>(tag->frameList("TIME").front());
```

Comment:
`timeframe` could be `nullptr` and should be checked in the next line with `if(timeframe && timeframe->data().size() >= 5) {`.